packages/components/stories/CollapsiblePanel.js

```diff
@@ -159,5 +227,29 @@ storiesOf('CollapsiblePanel', module)
 			<CollapsiblePanel{...propsCollapsiblePanelWithHeaderGroups} />
 			<p>Header with fixed circularProgress :</p>
 			<CollapsiblePanel {...propsCollapsiblePanelWithHeaderGroupsWithProgress} />
+			<p>Selected CollapsiblePanel:</p>
+			<CollapsiblePanel {...propsCollapsiblePanel} selected />
 		</div>
+	))
+	.addWithInfo('Selectable Panel', () => (
```

Comment:
Can you split :
* Key/value content
* Textual content
* Selected/Not Selected story